src/config/i18n/en/messages.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
same the comment in `src/config/i18n/en/messages.js` about having them as objects

Reply:
same as my previous reply 😂